x-pack/plugins/spaces/public/space_selector/space_selector.tsx

```diff
@@ -121,9 +122,13 @@ export class SpaceSelector extends Component<Props, State> {
         data-test-subj="kibanaSpaceSelector"
         panelled
       >
-        <KibanaPageTemplate.Section className="spcSpaceSelector__pageContent" color="transparent">
+        {/* Portal the fixed background graphic so it doesn't affect page positioning or overlap on top of global banners */}
+        <EuiPortal>
+          <div className="spcSelectorBackground" role="presentation" />
+        </EuiPortal>
+
+        <KibanaPageTemplate.Section color="transparent" alignment="center" paddingSize="xl">
```

Comment:
@elastic/kibana-security Can I get some feedback on the vertically-centered layout here? We used to render with a spacer and horizontally centered. Cee suggested the new layout(both v-centered and h-centered). I am somewhat indifferent - both look good to me.

Reply:
Thanks, Michael. I reverted the alignment back to top and put the spacer back in. 👍

Reply:
@jeramysoucy you may want to also remove the extra `paddingSize="xl"` I added to make up for the removed spacer - it's a lot of vertical height otherwise